core/state_processor.go

```diff
@@ -138,7 +137,7 @@ func applyTransaction(msg *Message, config *params.ChainConfig, gp *GasPool, sta
 
 	if tx.Type() == types.BlobTxType {
 		receipt.BlobGasUsed = uint64(len(tx.BlobHashes()) * params.BlobTxBlobGasPerBlob)
-		receipt.BlobGasPrice = eip4844.CalcBlobFee(*evm.Context.ExcessBlobGas)
+		receipt.BlobGasPrice = evm.Context.BlobBaseFee
```

Comment:
How come that the "gasprice" is calculated as the basefee? I mean, the base is only one part of the price, no?

Reply:
There is no blob priority fee yet, so it's just base fee!

Reply:
What was `ExcessBlobGas` exactly before? 
And the concept of `ExcessBlobGas` no longer exists?